source/iterators.tex

```diff
@@ -3985,11 +3985,11 @@
         operator-(const move_iterator& x, const move_sentinel<S>& y);
     friend constexpr iter_rvalue_reference_t<Iterator>
       iter_move(const move_iterator& i)
-        noexcept(noexcept(ranges::iter_move(i.current)));
+        noexcept(noexcept(ranges::iter_move(i.base())));
```

Comment:
This has normative impact: `i.base()` is potentially throwing, whereas `i.current` is not. The proposed change is equivalent to `noexcept(false)`.

Reply:
Good catch; reverted. Plus, I didn't want to mess with move_iterator anyway.
However, I'm left wondering why `base()` is not noexcept, given that no precondition is apparent.

Reply:
`base` returns a copy of `current`, which could potentially throw.